NullPointerException during create-runtime-image

NullPointerException experienced if the project artifact has not been created yet.

If you...
 - Skip the jar plugin (because the jar has been packed into a separate project)
 - create the image prior to assembling the jar (because it's being packed into the jar)
 - or other nonsense where the project lacks an artifact when moditect is invoked
...the build will fail on a NullPointerException.
Works fine in release 1.0.0.RC1

Caused by: java.lang.NullPointerException
    at org.moditect.mavenplugin.image.CreateRuntimeImageMojo.execute (CreateRuntimeImageMojo.java:126)
    at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo (DefaultBuildPluginManager.java:137)
    at org.apache.maven.lifecycle.internal.MojoExecutor.doExecute2 (MojoExecutor.java:370)
    ...

example pom:
```xml
<project xmlns="http://maven.apache.org/POM/4.0.0"
         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
    <modelVersion>4.0.0</modelVersion>

    <groupId>exmaple-sscce</groupId>
    <artifactId>moditect-image-npe-sscce</artifactId>
    <version>1.0.0</version>

    <build>
        <plugins>
            <plugin>
                <groupId>org.moditect</groupId>
                <artifactId>moditect-maven-plugin</artifactId>
                <version>1.0.0.Final</version>
                <executions>
                    <execution>
                        <id>generate-image</id>
                        <phase>generate-resources</phase>
                        <goals>
                            <goal>create-runtime-image</goal>
                        </goals>
                        <configuration>
                                <modulePath>
                                        <path>${project.build.directory}/modules</path>
                                </modulePath>
                                <modules>
                                <module>jdk.httpserver</module>
                                </modules>
                                <outputDirectory>
                                        ${project.build.directory}/jlink-image/slim-jre
                                </outputDirectory>
                        </configuration>
                    </execution>
                </executions>
            </plugin>
        </plugins>
    </build>
</project>
```

--- a/.github/workflows/release.yml
+++ b/.github/workflows/release.yml
@@ -82,14 +82,14 @@ jobs:
           export GPG_TTY=$(tty)
           ./mvnw -ntp -B --file pom.xml \
             -Drepository.url=https://${{ github.actor }}:${{ secrets.GITHUB_TOKEN }}@github.com/${{ github.repository }}.git \
-            -Dmaven.site.skip=true -Drelease=true deploy
+            -Dmaven.site.skip=true -Ppublication,stage
 
       - name: Release to GitHub
         env:
           JRELEASER_GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           JRELEASER_BRANCH: ${{ env.BRANCH }}
         run: |
-          ./mvnw -B --file pom.xml -pl :moditect-parent -Pjreleaser jreleaser:release
+          ./mvnw -ntp -B --file pom.xml -pl :moditect-parent -Pjreleaser jreleaser:release
 
       - name: JReleaser output
         if: always()

--- a/core/pom.xml
+++ b/core/pom.xml
@@ -29,6 +29,8 @@
     <artifactId>moditect</artifactId>
     <packaging>jar</packaging>
     <name>ModiTect</name>
+    <url>https://github.com/moditect/moditect</url>
+    <description>Module system tooling</description>
 
     <dependencies>
         <dependency>

--- a/maven-plugin/pom.xml
+++ b/maven-plugin/pom.xml
@@ -28,6 +28,8 @@
     <artifactId>moditect-maven-plugin</artifactId>
     <packaging>maven-plugin</packaging>
     <name>ModiTect Maven Plug-In</name>
+    <url>https://github.com/moditect/moditect</url>
+    <description>ModiTect Maven Plug-In</description>
 
     <dependencyManagement>
         <dependencies>

--- a/parent/pom.xml
+++ b/parent/pom.xml
@@ -172,6 +172,15 @@
 
     <profiles>
         <profile>
+            <id>stage</id>
+            <properties>
+                <altDeploymentRepository>local::file:${rootdir}/target/staging-deploy</altDeploymentRepository>
+            </properties>
+            <build>
+                <defaultGoal>deploy</defaultGoal>
+            </build>
+        </profile>
+        <profile>
             <id>jreleaser</id>
             <build>
                 <plugins>
@@ -190,6 +199,24 @@
                                         <documentation>https://github.com/moditect/moditect</documentation>
                                     </links>
                                 </project>
+                                <signing>
+                                    <active>ALWAYS</active>
+                                    <armored>true</armored>
+                                </signing>
+                                <deploy>
+                                    <maven>
+                                        <nexus2>
+                                            <maven-central>
+                                                <active>ALWAYS</active>
+                                                <url>https://oss.sonatype.org/service/local</url>
+                                                <snapshotUrl>https://oss.sonatype.org/content/repositories/snapshots/</snapshotUrl>
+                                                <closeRepository>true</closeRepository>
+                                                <releaseRepository>true</releaseRepository>
+                                                <stagingRepositories>target/staging-deploy</stagingRepositories>
+                                            </maven-central>
+                                        </nexus2>
+                                    </maven>
+                                </deploy>
                                 <release>
                                     <github>
                                         <tagName>{{projectVersion}}</tagName>

